### ISSUE TITLE
Image Labels

### ISSUE THREAD

**Issue by psavery** [NERSC/shifter#300]
Hi there,

I was wondering if shifter images store labels from their corresponding docker images, and if there is a way to look the labels up? For docker and singularity, we are storing important metadata about the images in the labels, and we'd like to do the same with shifter if possible.

For instance, take this Dockerfile (from [psavery/test:latest](https://hub.docker.com/layers/psavery/test/latest/images/sha256-4bbfc7af53daa46e3a2e2debdc7391d6223e720c47140f410446f9fac552417e?context=explore)):
```Dockerfile
FROM ubuntu:20.04
  
LABEL label=3 \
  nested.label=True
```

We can inspect the labels with docker like so:
```
docker inspect --format='{{json .Config.Labels}}' psavery/test:latest
# {"label":"3","nested.label":"True"}
```

Or similarly in singularity, if the labels are defined in the `.def` file:

```
singularity inspect test.simg 
# label: 3
# nested.label: True
```

If labels are not available in shifter, could they be added?

**Comment by scanon** [comment]
Currently no.  Is the main use case similar to what you show before?  Just being able to inspect a specific image and get the labels?  You don't, for example, need to query based on these do you?

**Comment by psavery** [comment]
@scanon That is correct, we just need to inspect a specific image and get the labels. We don't need to query based on them.

**Comment by scanon** [comment]
I think this could be added without too much trouble.  I'm going to turn this into a feature request.

**Comment by psavery** [comment]
@scanon Any updates on when this might be implemented?

**Comment by scanon** [comment]
Let me see if I could do a quick version.

**Comment by scanon** [comment]
Fixed.
